benchmark/README.md

```diff
@@ -0,0 +1,14 @@
+# Benchmark
+
+The following graphs show the benchmark results (runtime (in secs)) of running match attempts, using different PSI algorithms, on various Google cloud general-purpose virtual machines (VMs) like n2-standard-32, n2-standard-48, n2-standard-64 and n2-standard-80. For each experiment, same type of VMs are used for both sender and receiver. The match attempts are performed between a fixed receiver dataset of 50m records and variety of sender datasets of 50m, 100m, 150m, 200m and 250m records each. 
```

Comment:
Add/embed a link to the machines to show how many cores/ram each machine type has.

Reply:
I'm also wondering, if the opposite scenario is more appropriate, since a receiver of a match attempt usually have a larger dataset.

Reply:
I'm also wondering if the opposite scenario is more appropriate since the receiver of a match attempt usually has a larger dataset.

Reply:
We can discuss this in today's weekly sync-up.